gyp_xwalk

```diff
@@ -354,6 +354,18 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  parameter = '-Dverify_xwalk_apk='
```

Comment:
To me it makes more sense to set (and document) this in `build/common.gypi` instead like a regular variable.